lib/airbrake-ruby/notice.rb

```diff
@@ -240,6 +242,15 @@ def extract_custom_attributes(exception)
         )
       end
     end
+
+    def deep_dup(params, level = 1)
+      level = 0 if level > 3
```

Comment:
That'll work.

I spotted more serious issues with the approach (test failures).